understanding/22/focus-not-obscured-minimum.html

```diff
@@ -48,14 +53,95 @@ <h2>Intent of Focus Not Obscured (Minimum)</h2>
+            <figcaption>Placeholder for future illustrations showing an abstract representation of an accordion pushing content down when opened</figcaption>
+        </figure>
+</li>
+
+          <li><strong>When the additional content appears, existing content reflows.</strong> The popout sidebar on the <a href="https://www.w3.org/TR/WCAG22/">WCAG standard</a> is an example of this pattern. When the side menu is activated, it discloses a new section of information along the left side of the page. The main content area is reduced horizontally to accommodate the new content, and the existing content reflows to fit in the thinner space. As a result, there is no overlapping content between the two sections; the item receiving focus, whether in the left navigation or in the main content, will not be obscured by the other section.</ br>
+
+            <figure id="">
+              <img src="" alt="" />
+              <figcaption>Placeholder for future illustration pair showing an abstract representation of a sidebar reflowing a thinner main content region</figcaption>
+          </figure>
+         </li>
+          <li><strong>When the additional content is disclosed, it takes focus and the tab ring is constrained to the new content until it is dismissed.</strong> This modality is somewhat like a dialog, in that a user cannot navigate beyond the disclosed content by keyboard without dismissing it first (typically by pressing Esc). However, unlike in a modal dialog, in some implementations a pointer user may be able to interact with content outside the disclosed section without dismissing it. Since this pattern potentially creates an inequitable experience between keyboard and pointer users, it should be used cautiously. That said, it does prevent the disclosed content from obscuring the keyboard focus in the main content, and thus should pass this SC. This is described and demonstrated in a short video in the Knowbility article in the reference section, under the section heading <a href="https://knowbility.org/blog/2020/accessible-slide-menus#keep-keyboard-focus-in-the-slide-out-navigation-until-its-closed">Keep keyboard focus in the slide-out navigation until it’s closed</a>.</ br>
+
+            <figure id="">
+              <img src="" alt="" />
+              <figcaption>Placeholder for future illustrations showing an abstract representation of a side panel where the tab ring is restricted to the panel</figcaption>
+          </figure>
+         </li>
+          <li><strong>The disclosure expands into an area of the page containing no other content.</strong> Many pages are designed with wide margins, providing significant white space into which new content can be disclosed. Many chatbots and toasters are designed to 'slide up' into the right unpopulated side of a page. Where authors are careful to ensure content is not obscured at each breakpoint in a responsive design, no obscuring of other operable content need occur.</ br>
```

Comment:
is the `</ br>` necessary here?  if so, it should instead be `<br>`

Reply:
suggest "toast notifications" rather than "toasters"

re:
>...into the right unpopulated side of a page...

maybe remove "the right" and replace with "an".  I've seen it on both sides (i believe github even does lower left) - and i'd assume this not necessarily the typical location for right to left language pages.